src/main/java/net/aufdemrand/denizen/events/player/PlayerPlacesBlockScriptEvent.java

```diff
@@ -98,13 +98,20 @@ public ScriptEntryData getScriptEntryData() {
     }
 
     @Override
-    public HashMap<String, dObject> getContext() {
-        HashMap<String, dObject> context = super.getContext();
-        context.put("location", location);
-        context.put("material", material);
-        context.put("cuboids", cuboids);
-        context.put("item_in_hand", item_in_hand);
-        return context;
+    public dObject getContext(String name) {
+        if (name.equals("location")) {
+            return location;
+        }
+        else if (name.equals("material")) {
+            return material;
+        }
+        else if (name.equals("cuboids")) {

```

Comment:
add "// Deprecated in favor of context.location.cuboids" to the end

`else if (name.equals("cuboids")) { // Deprecated in favor of context.location.cuboids`